valhalla/mjolnir/graphtilebuilder.h

```diff
@@ -256,6 +257,17 @@ class GraphTileBuilder : public baldr::GraphTile {
                        bool& added,
                        bool diff_names = false);
 
+  /**
+   * Set the OSM id for the most recently added node.
+   */
+  void set_last_node_osmid(const uint64_t id) {
```

Comment:
why not something just like the the nodes and edges:

`std::vector<uint64_t>& osm_node_ids();`

Reply:
Happy to do that, especially given that it is consistent with nodes and edges. I guess this feels to me like it hides more implementation detail and the existence of a method with this name tells us something.

Reply:
Changed.